src/wasm/wasm-emscripten.cpp

```diff
@@ -334,41 +334,44 @@ inline void exportFunction(Module& wasm, Name name, bool must_export) {
   wasm.addExport(exp);
 }
 
+void EmscriptenGlueGenerator::generateDynCallThunk(std::string sig) {
+  auto* funcType = ensureFunctionType(sig, &wasm);
+  if (!sigs.insert(sig).second) {
+    return; // sig is already in the set
+  }
+  Name name = std::string("dynCall_") + sig;
+  if (wasm.getFunctionOrNull(name) || wasm.getExportOrNull(name)) {
```

Comment:
just checking for an export is enough, isn't it? I'd remove the function check, as it doesn't help, and can be confusing if we change things in the future.

Reply:
I don't know, this change only moves this code, doesn't modify it.  So I'd rather do that as a followup if that ok?

Reply:
Oh ok, I didn't realize it was just moved.